galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.cloud.manager/src/main/java/dev/galasa/cloud/CloudContainer.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package dev.galasa.cloud;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Cloud Container
+ * 
+ * The Cloud Container provides the ability to integration test applications across one or more cloud platforms.   It is not intended to 
+ * allow deep testing of a container, for that level of testing you should use a dedicated container manager like the Docker, Kubernetes or Podman(to be written) Managers.
+ * 
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.FIELD })
+public @interface CloudContainer {
```

Comment:
should this not be annotated CloudManagerField?

Reply:
it was,  dont know where it went